scripts/framework-applications/notify-successful-suppliers-for-framework.py

```diff
@@ -66,9 +68,17 @@
 
     prefix = "[Dry Run] " if DRY_RUN else ""
 
+    # Add in any framework-specific dates etc here
+    extra_template_context = {
+        "intentionToAwardAt_dateformat": "12 September 2020",
+        "frameworkLiveAt_dateformat": "28 September 2020",
+    }
```

Comment:
@katstevens If we have to hardcode these dates should we add a TODO to update them for the next framework?

Reply:
Sure - a TODO will make it a bit more obvious.